chart/kubeapps/values.yaml

```diff
@@ -1515,7 +1515,7 @@ clusters:
     domain: cluster.local
 ## @param featureFlags [object] Feature flags (used to switch on development features)
 ##
-featureFlags:
+featureFlags: ""
```

Comment:
This should be `{}`, but the Bitnami team is already aware and they're gonna fix it in a separate upstream PR

Reply:
Well, they didn't perform the change (probably they're on PTO), so let me squeeze the change in our dev chart so that it get synced with the upstream one at some point.